core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -18,8 +18,6 @@
 import static org.stellar.anchor.util.SepHelper.validateAmount;
 import static org.stellar.anchor.util.SepHelper.validateAmountLimit;
 import static org.stellar.anchor.util.SepLanguageHelper.validateLanguage;
-import static org.stellar.anchor.util.StringHelper.isEmpty;
-import static org.stellar.sdk.xdr.MemoType.MEMO_NONE;
 
 import io.micrometer.core.instrument.Counter;
 import jakarta.transaction.Transactional;
```

Comment:
Jakarta is old. Should we use `javax.transaction.Transactional`?

Reply:
`javax` was renamed to `jakarta` https://mvnrepository.com/artifact/javax.transaction/jta